views/content/approval.html

```diff
@@ -12,22 +12,23 @@
 <h1 class="base--h1">{{title}}</h1>
 
 {% if revision %}
-  <h2 class="base--h2">{{config.actions.approve}} :: [TBAdded: Primary Key Contents]</h2>
+  <h2 class="base--h2">{{config.actions.approve|capitalize}} :: [TBAdded: Primary Key Contents]</h2>
 
   {% if data %}
+    <div class="listing">
     {% for attr, inputs in data %}
-      <h3>{{attr}}</h3>
+      <h3 class="listing--h3">{{attr|capitalize}}</h3>
 
       {% if inputs.length %}
         {# repeatable content #}
         {% for rpt in inputs %}
           {% for input, value in rpt %}
             {% if loop.first %}
-              <dl class="base--dl">
+              <dl class="listing--dl">
```

Comment:
Might be easier to revisit once we have repeatables actually in here. Should we table this?